sdk/storage/storage-blob/src/BatchResponseParser.ts

```diff
@@ -146,6 +145,9 @@ export class BatchResponseParser {
 
       if (contentId != NOT_FOUND) {
         deserializedSubResponse._request = this.subRequests.get(contentId)!;
+        deserializedSubResponses[contentId] = deserializedSubResponse;
```

Comment:
So the `contentId` is in the valid range? Could you please add a comment about it here?  Any test to verify the ordering?

Reply:
The content id is passed from request to the response. In our SDK, we set it to an index in the request. Will add a check here anyway.

All our test cases on the blob batch assume the ordering is guaranteed and they never break. So the service may already keep the order but there is no documentation on this.